Sources/ImageRequest.swift

```diff
@@ -264,34 +264,34 @@ public extension ImageRequest {
 
     /// Appends a processor to the request. You can append arbitrary number of
     /// processors to the request.
-    public func processed<P: ImageProcessing>(with processor: P) -> ImageRequest {
+    func processed<P: ImageProcessing>(with processor: P) -> ImageRequest {
         var request = self
         request.process(with: processor)
         return request
     }
 
     /// Appends a processor to the request. You can append arbitrary number of
     /// processors to the request.
-    public mutating func process<Key: Hashable>(key: Key, _ closure: @escaping (Image) -> Image?) {
+    mutating func process<Key: Hashable>(key: Key, _ closure: @escaping (Image) -> Image?) {
         process(with: AnonymousImageProcessor<Key>(key, closure))
     }
 
     /// Appends a processor to the request. You can append arbitrary number of
     /// processors to the request.
-    public func processed<Key: Hashable>(key: Key, _ closure: @escaping (Image) -> Image?) -> ImageRequest {
+    func processed<Key: Hashable>(key: Key, _ closure: @escaping (Image) -> Image?) -> ImageRequest {
         return processed(with: AnonymousImageProcessor<Key>(key, closure))
     }
 }
 
 internal extension ImageRequest {
     struct CacheKey: Hashable {
         let request: ImageRequest
-
-        var hashValue: Int {
+        
+        func hash(into hasher: inout Hasher) {
             if let customKey = request._ref.cacheKey {
-                return customKey.hashValue
+                hasher.combine(customKey)
```

Comment:
It seems that this changes the behavior of `CacheKey`, please add return or `else`.

Reply:
Whoops! Good catch. Also updated `LoadKey`, had the same issue.